controllers/managedocs_controller.go

```diff
@@ -87,7 +88,7 @@ const (
 	deployerCSVPrefix                  = "ocs-osd-deployer"
 	ocsOperatorName                    = "ocs-operator"
 	mcgOperatorName                    = "mcg-operator"
-	egressNetworkPolicyName            = "egress-rule"
+	egressPolicyName                   = "default"
```

Comment:
- nit, some formatting error here? an extra tab 🤔

Reply:
No, we have to change the PolicyName form egress-rule to default as EgressFirewall doesn't accept any other name. they have a check - ```Invalid value: "egress-rule": metadata.name in body should match '^default$'" for field "metadata.name```

Reply:
@rewantsoni that will create a backward compatibility issue for existing deployments, as it will leave non-reconciled existing  egressNetworkPolicy resources on upgraded clusters.

Please use the original name for `egressNetworkPolicy` resource and the name default for `EgressFirewall` resources

Reply:
Done

Reply:
ack